includes/wc-payment-api/class-wc-payments-api-client.php

```diff
@@ -183,6 +183,7 @@ public function create_charge( $amount, $source_id ) {
 	 * @param bool   $off_session                     - Whether the payment is off-session (merchant-initiated), or on-session (customer-initiated).
 	 * @param array  $additional_parameters           - An array of any additional request parameters, particularly for additional payment methods.
 	 * @param array  $payment_methods                 - An array of payment methods that might be used for the payment.
+	 * @param string $cvc_confirmation The CVC confirmation for this payment method.
```

Comment:
Nit: This line is misaligned with the other parameter descriptions.

Reply:
Fixed on d8b4cfa.